### ISSUE TITLE
Biodiversity Data: Using RDF,Github and PSPS to provide accessible RDF data about species

### ISSUE THREAD

**Issue by retog** [zazuko/knowledge-graph-forum#6]
# What would you like to talk about

I would like to show some of the tools we use at [plazi](http://plazi.org) to make RDF data accessible to humans and machines. Especially the new effort to provide vernacular names on https://vernacular.plazi.org/ and used tool [PSPS](https://github.com/factsmission/psps) that provides somethinig similar to GitHub pages but for RDF data but also the main corpus of more than 300'000 treatments extracted from 28'000 scientific articles that describe 250'000 biological species (taxa).

# Who might be interested in that (technical audience, management, data scientists, etc.)

Basically anybody who thinks in triples and breathes RDF.

# What would you like to know from other participants, what feedback are you looking for

Ideas on how to improve the tool and possible other applications

# What is your background (name, affiliation, etc)

I work with plazi for the swiss linked data company factsmission.com. To lear more, click on my face.

**Comment by ktk** [comment]
Thanks for the submission, we have enough slots for all proposals so far. Please contact me at adrian.gschwend@zazuko.com for a ticket voucher. Please also make sure that you have travel arrangements in case you come in from further away. We will register your session at the DINAcon page.

Last but not least we will create a speaker-page for you, unless you do not want to have that. It will look like [this](https://dinacon.ch/speakers/bart-van-leeuwen/) for example. You are welcome to send me more information about you on the email above, also including a picture if you like.

Looking forward to have you at the conference!
